api/src/pcapi/domain/music_types.py

```diff
@@ -325,7 +326,7 @@ class MusicType:
 }
 MUSIC_SUB_TYPES_BY_CODE = {
     music_sub_type.code: music_sub_type for music_type in music_types for music_sub_type in music_type.children
-}
+}  # WARNING: for code -1, the sub type is not unique, it returns the one with the one with the slug CHANSON_VARIETE-OTHER
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Je comprend pas trop ce commentaire :/ 

aussi, 
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** `the one with` est répété‏